src/test-extension-exemptions-workspace/test-extension-exemptions-workspace.service.spec.ts

```diff
@@ -142,7 +149,7 @@ describe('TestExtensionExemptionsWorkspaceService', () => {
         errored = true;
       }
 
-      expect(errored).toBe(true);
```

Comment:
Shouldn't errored === true here since that's the point of the test?

Reply:
You're right, thank you
![Screenshot 2023-01-25 074236](https://user-images.githubusercontent.com/112484102/214566717-8e94a131-0af1-43ec-b343-142c4f96b65d.png)